lab_1/start.py

```diff
@@ -15,11 +16,31 @@
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'de.txt'), 'r', encoding='utf-8') as file_to_read:
         de_text = file_to_read.read()
 
+    with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'la.txt'), 'r', encoding='utf-8') as file_to_read:
+        la_text = file_to_read.read()
+
     with open(os.path.join(PATH_TO_TEXTS_FOLDER, 'unknown.txt'), 'r', encoding='utf-8') as \
             file_to_read:
         unknown_text = file_to_read.read()
 
-    EXPECTED = 'en'
-    RESULT = ''
-    # DO NOT REMOVE NEXT LINE - KEEP IT INTENTIONALLY LAST
-    assert RESULT, 'Detection not working'
+    unk = main.create_language_profile('unk', unknown_text, [])
+    de = main.create_language_profile("de", de_text, [])
+    en = main.create_language_profile("en", en_text, [])
+    la = main.create_language_profile("la", la_text, [])
+    main.save_profile(unk)
+
+    de_profile = main.load_profile(os.path.join(PATH_TO_LAB_FOLDER, r'profiles\de.json'))
+    en_profile = main.load_profile(os.path.join(PATH_TO_LAB_FOLDER, r'profiles\en.json'))
+    la_profile = main.load_profile(os.path.join(PATH_TO_LAB_FOLDER, r'profiles\la.json'))
+    unk_profile = main.load_profile(os.path.join(PATH_TO_LAB_FOLDER, r'profiles\unk.json'))
+
+    languages = ['en', 'de', 'la']
+    profiles = [en_profile, de_profile, la_profile]
+    RESULT = main.detect_language_advanced(unk_profile,profiles ,languages , 5)
```

Comment:
can you describe in writing what happens in `start.py`?

Reply:
import function from main
read text from files that we have to variables in type string
the main aim is to detect unkown language using profiles(*.json) and texts.
we have 2 ways how we can do it:
-by our own
-using external  profiles
with the help of texts we create  language profiles(function create_language_profile) and then detect language(function detect_language_advanced). this is result numer 1
then we load external profiles(function load_profile) and again detect(function detect_language_advanced).this is result number 2
and finally we compare results: if they are identical, everything works great , however, if there is 'Detection not working' something doesn't work)))
that's all